### ISSUE TITLE
First podman invocation: Error: overlay test mount with multiple lowers failed, but succeeded with a single lower

### ISSUE THREAD

**Issue by edsantiago** [containers/podman#10248]
On Ubuntu-old (with cgroups v1 and runc; possibly with other weird settings), the very very first time rootless podman is invoked, it emits a level=error message (but succeeds with exit 0 regardless):
```console
$  podman version
time="2021-05-06T13:13:04Z" level=error msg="overlay test mount with multiple lowers failed, but succeeded with a single lower"
Version:      3.2.0-dev
...
```
I can't reproduce on RHEL 8.4, possibly because I can't get a recent-enough version of podman. This looks like a very new error.

The message does not seem to happen after the first invocation of podman.

**Comment by edsantiago** [comment]
[Real-world failure](https://storage.googleapis.com/cirrus-ci-6707778565701632-fcae48/artifacts/containers/podman/5963423050629120/html/sys-podman-ubuntu-2010-rootless-host.log.html). And [another one](https://storage.googleapis.com/cirrus-ci-6707778565701632-fcae48/artifacts/containers/podman/4815796649066496/html/sys-podman-ubuntu-2010-rootless-host.log.html).

**Comment by Luap99** [comment]
I think this was already fixed by @rhatdan

**Comment by rhatdan** [comment]
Fixed in containers storage c36df2f04523a38b2edda2445a899bc63d79448d

**Comment by rhatdan** [comment]
https://github.com/containers/storage/pull/885
